tests/test_table_transformations.py

```diff
@@ -423,6 +473,29 @@ def test_result_data_shape_patient_diagnosis_radiology_study(self):
         expected_df.set_index(['Subject Id', 'Diagnosis Id', 'Radiology Id', 'Study Id'], inplace=True)
         pdt.assert_frame_equal(df, expected_df, check_dtype=False, check_categorical=False, check_like=True)
 
+    def test_result_data_shape_study_only(self):
+        test_data = [
+            ['P1', 'Study1', 'Study.title', '\\Study\\Title\\', 'Study title',  None, 'Study 1', 1, 'TEST'],
+            ['P1', 'Study2', 'Study.title', '\\Study\\Title\\', 'Study title', None, 'Study 2', 1, 'TEST'],
+            ['P1', 'Study1', 'IndividualStudy.individual_study_id', '\\IndividualStudy\\individual_study_id\\', 'Individual study id', None, '1', 1, 'TEST'],
+            ['P1', 'Study2', 'IndividualStudy.individual_study_id', '\\IndividualStudy\\individual_study_id\\', 'Individual study id', None, '2', 1, 'TEST']
+        ]
+        observations_df = pd.DataFrame(test_data, columns=['patient.subjectIds.SUBJ_ID',
+                                                           'Study',
+                                                           'concept.conceptCode', 'concept.conceptPath',
+                                                           'concept.name', 'numericValue',
+                                                           'stringValue', 'patient.id', 'study.name'])
+
+        df = transform_obs_df(observations_df)
+
+        self.assertIsNotNone(df)
+        expected_df = pd.DataFrame([
+            ['P1', 'Study1', 'Study 1', '1'],
+            ['P1', 'Study2', 'Study 2', '2'],
+        ], columns=['Subject Id', 'Study Id', 'Study title', 'Individual study id'])
+        expected_df.set_index(['Subject Id', 'Study Id'], inplace=True)
+        pdt.assert_frame_equal(df, expected_df, check_dtype=False, check_categorical=False, check_like=True)
+
```

Comment:
Within a class there typically is a single empty line between functions.

Reply:
ah yes, thanks for noticing, I will fix it